src/components/ciPipeline/SourceMaterials.tsx

```diff
@@ -13,11 +13,13 @@ import { SourceMaterialsProps } from './types'
 import InfoColourBar from '../common/infocolourBar/InfoColourbar'
 import { ReactComponent as InfoIcon } from '../../assets/icons/info-filled.svg'
 import { reactSelectStyles } from '../CIPipelineN/ciPipeline.utils'
+import { ConditionalWrap } from '../common'
+import Tippy from '@tippyjs/react'
 
 export const SourceMaterials: React.FC<SourceMaterialsProps> = function (props) {
     const [isProviderChanged, setProviderChanged] = useState(false)
-    const { ciPipelineId } = useParams<{ ciPipelineId: string }>()
     const isMultiGit = props.materials.length > 1
+    const islinkedCI = window.location.href.indexOf('linked-ci') != -1
```

Comment:
instead of != please do it !==

Reply:
Here -1 isn't representing anything please move it to a constant and name it according to the usuage

Reply:
or can we use includes instead of indexOf?